api/nakadi-event-bus-api.yaml

```diff
@@ -484,3 +860,119 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Event'
+
+  EventType:
+    description: An event type defines the schema and its runtime properties.
+    properties:
+      name:
+        type: string
+        description: |
+          Name of this EventType. The name for the EventType must follow the pattern
+          'organization:team:application:event-type'
+      topic:
+        type: string
+        description: |
+          References the name of the low-level topic that is holding the stream of Events for this EventType.
+          Usually is the same as the name property.
```

Comment:
Maybe we should say, that the topic field will be provided, only if it differs from the `EventType.name` value